xscen/catalog.py

```diff
@@ -496,18 +529,18 @@ def create(
         cls,
         filename: Union[os.PathLike, str],
         *,
-        project: Optional[dict] = None,
-        overwrite=False,
+        project: dict = None,
+        overwrite: bool = False,
     ):
         r"""Create a new project catalog from some project metadata.
 
         Creates the json from default :py:data:`esm_col_data` and an empty csv file.
 
         Parameters
         ----------
-        filename : PathLike
+        filename : os.PathLike or str
           A path to the json file (with or without suffix).
-        project : dict-like
+        project : dict, optional
```

Comment:
I'm confused, can `project` be `None` or not?

Reply:
This had been changed to be in line with my statement on Optional, but I've put it back everywhere.